.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
There is also can be `indent_size = N` (for alignment, [article about this](https://dmitryfrank.com/articles/indent_with_tabs_align_with_spaces)), but I don't know the preferred size (and I didn't see alignment of code in this project at all).